module-user/src/main/kotlin/com/example/user/application/service/SelectStreamerUserService.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.user.application.service
+
+import com.example.user.application.port.`in`.SelectStreamerUserUseCase
+import com.example.user.application.port.out.StreamerUserRepository
+import com.example.user.domain.model.StreamerUser
+import com.example.user.domain.model.StreamerUserStatus
+
+class SelectStreamerUserService(private val streamerRepository: StreamerUserRepository) : SelectStreamerUserUseCase {
+
+    override fun selectPendingStreamers() : List<StreamerUser> {
+        return streamerRepository.findAllByStatus(StreamerUserStatus.PENDING)
+            .ifEmpty { emptyList() }
```

Comment:
```suggestion
        return streamerRepository.findAllByStatus(StreamerUserStatus.PENDING)
```

streamerRepository.findAllByStatus()의 반환형이 이미 List<StreamerUser> 입니다,
따라서 emptyList()를 명시적으로 반환할 필요는 없다고 생각합니다.